cgnsutilities/cgnsutilities.py

```diff
@@ -127,26 +135,52 @@ def printInfo(self):
         print("Wall Boundary Cells:", boundaryCells)
         print("Wall Boundary Nodes:", boundaryNodes)
 
-    def printBlockInfo(self):
-        """Print some information on each block to screen.
-        This info can be helpful assessing overset meshes"""
+    def getBlockInfo(self):
```

Comment:
Just for my understanding, is the primary purpose here to provide block information to other codes (like for testing/comparing in the pypyh PR) or is there a separate function (beside printing)?

Reply:
My motivation was the pyHyp PR, but I can see this being useful in general to access the block information from Python.